vspace/src/main/java/edu/asu/diging/vspace/config/ExhibitionLanguageConfig.java

```diff
@@ -0,0 +1,58 @@
+package edu.asu.diging.vspace.config;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.AbstractEnvironment;
+import org.springframework.core.env.Environment;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.stereotype.Component;
+
+import edu.asu.diging.vspace.core.model.impl.ExhibitionLanguage;
+
+@Component
+@PropertySource(value= "classpath:exhibitionLanguages.properties" , factory=JsonPropertySourceFactory.class)
+@Configuration
+public class ExhibitionLanguageConfig {
+
+    @Autowired
+    private Environment environment;
+    
+    List<Map>  exhibitionLanguageList = new ArrayList<Map>();
+    
+    /**
+     * Fetches the configured language list from environment property source and stores in exhibitionLanguageList
+     * 
+     */
+    @PostConstruct
+    public void init() {
+        for(Iterator it = ((AbstractEnvironment) environment).getPropertySources().iterator(); it.hasNext(); ) {
```

Comment:
since only one is listed in the `PropertySource`, can there be more in the list than just this one?

Reply:
Didn't get you. There are multiple property sources. we are iterating through it to search for the particular property we need

Reply:
Are there more than the one listed in the `@PropertySource` annoation?

Reply:
Yes, there are around 10 in the list.